google/ggrc-core
# [Scope Sync] GGRC-7855 Migrate Primary/Secondary contract to Compliance Contacts

PR should be merge in **ggrcq_ggrc_scope_sync** branch

# Issue description
Migrate Primary/Secondary contract to Compliance Contacts for Scope objects

# Steps to test the changes
Create tests data before migration for Primary/Secondary contacts
Run migration
Check that Compliance Contacts were populated correctly 

# Solution description
Add migration for union Primary/Secondary Contacts and move to Compliance Contacts

# Sanity checklist


# Migration checklist

# PR Review checklist
